manual/manual.adoc

```diff
@@ -380,33 +385,15 @@ normal damage until the end of level.
 image:../sprites/pinsa0.png[Invisibility Cloak] |
 Makes you almost invisible for a limited time. Monsters still detect
 your presence, but they'll find it much harder to aim.
+| **Negentropic Surge** +
+image:../sprites/megaa0.png[Negentropic Surge] |
+Maxes you out to 200% health and armor.
 | **Vanguard Device** +
 image:../sprites/pinva0.png[Vanguard Device] |
 Makes you immune to all damage for a limited time, allowing you to get
 past overwhelming defences and unavoidable traps.
 |==========================
 
-=== Color blind accessibility
```

Comment:
Can we keep this as a separate section please?

Reply:
Still thinking about this.

On the one hand, I stand by my view that splitting the key information is not good.

On the other, I *do* think that "virtue signalling" is actually helpful and works well with the new "free as in freedom" section - especially to clarify that we're not *that* kind of fossbro outfit.

Maybe we can just add a blurb about accessibility features and considerations generally...

Reply:
something like this, just before the "freedoom is about freedom" section:
> === Accessibility
> 
> Freedoom is a volunteer project that receives contributions from a wide variety of people with differring abilities and needs. The community tries to address what accessibility issues there may be with Freedoom but there is always room for improvement.
> 
> Some areas that have received attention as of fall 2023 are described below. This remains a work in progress and the list is incomplete, and contributions and feedback are welcome:
> 
> * **Color blind accessibility:** Graphics are selected so that nothing is distinguished solely by hue. Each key sprite has different markings, armour designs have different silhouettes, and switches have up/down positions; where color is the main difference, it always accompanies a change in brightness as well. (The pain brothers are confusingly similar even for those who are not color blind. :))
> 
> * **Controls:** Areas that can only be accessed by straferunning are discouraged, as it may not be available on certain controller configurations.
> 
> * **Cognitive:** *<okay i know for a fact that i've had a bunch of shit merged
> that was primarily motivated by accommodating my ADHD but i can't remember any particulars at the moment>* ...

Reply:
I went ahead and added it.

Reply:
I'm sorry but I really dislike this change.

I think your take on this being about "virtue signalling" is completely off base. The text here serves a specific practical purpose - there is a visual language in play that is non obvious (eg. the way the horns point matching the visual shapes), and the purpose off this section is to explain it. It's nothing to do with being performative or boastful about how accessible we are. This is exactly the kind of thing that *needs* to be documented in the manual. As it currently stands, you've almost deleted that entirely.

I'd be curious to see if anyone else got the same impression about this section because I'm not understanding this take at all. I read through the text again several times and I don't get it. Maybe I'm blind to it because I wrote it. It certainly was never my intention.

I don't like the current version because the new accessibility section really is just performative and I don't think adds anything substantial that needs to be documented.

Anyway. The core of the change as it stands is that it merges the part about color blindness into the section with the keys. There's certainly a reasonable argument for doing this - the two are logically connected. The reasons I think this should be its own section are:

1. I don't like bloating the keys section by adding too much text. Putting too much text inside tables undermines the visual benefit of placing information into tables, plus I think it's just bad style in general.
2. Having a dedicated section makes the information more discoverable because the section has a heading that catches the eye. If you're color blind and you scan through the text, you can notice the heading "color blind accessibility", and think "yes, I'm color blind, I should probably read this". For the same reason, I think it's important that we explicitly use the term "color blind", which your change removes.
3. Having a dedicated section allows us to use illustrations to demonstrate the visual language that's used. There is an illustration in the section as it stands; it could probably be improved.

Reply:
Oops, didn't realize I'd completely removed the words "color blind".

I'll add the original section back but will still keep a mention in the key section itself since it is helpful information even for people who aren't colour blind, or whether or not they're colour blind might just happen not to be cognitively able to go all the way to the end of the section.

> I don't like the current version because the new accessibility section really is just performative and I don't think adds anything substantial that needs to be documented.

That new section was actually because I thought about that "virtue signalling" comment again, decided against it and ended up swinging in the opposite direction. For what it's worth, writing that new section was actually what got me started on #1104.